src/ui/.prettierignore

```diff
@@ -0,0 +1,8 @@
+/node_modules
```

Comment:
Double checking how these are supposed to resolve with the / and whether those are the root of the project, root of the users namespace, or root relative to prettierignore. I believe node_modules will be at the root of the project, but public is next to prettierignore